opencl/opencl-aot/source/main.cpp

```diff
@@ -477,7 +504,19 @@ int main(int Argc, char *Argv[]) {
         {sse42, "corei7"},
         {avx, "corei7-avx"},
         {avx2, "core-avx2"},
-        {avx512, "skx"}};
+        {avx512, "skx"},
+        {wsm, "corei7"},
+        {snb, "corei7-avx"},
+        {ivyb, "corei7-avx"},
+        {bdw, "core-avx2"},
+        {cfl, "core-avx2"},
+        {adl, "core-avx2"},
+        {skylake, "core-avx2"},
+        {skx, "skx"},
+        {clk, "skx"},
```

Comment:
```suggestion
        {clk, "cascadelake"},
```

Reply:
Unresolving conversation, it wasn't fixed